### ISSUE TITLE
React admin toolbar wrong color

### ISSUE THREAD

**Issue by eeriksp** [jhipster/generator-jhipster#9675]
##### **Overview of the issue**

I generated a new project with Kotlin blueprint and React frontend, ran it on localhost and discovered that that both the main toolbar as well as the buttons on it were white as seen in the picture:
![image](https://user-images.githubusercontent.com/35540617/57223350-694cba80-700e-11e9-9331-02ab94ba2bbe.png)

##### **Motivation for or Use Case**

Because both the toolbar and the buttons were white, it was almost impossible the see the buttons.

##### **Reproduce the error**

Generate a new project using React frontend and kotlin:
```
jhipster --blueprint kotlin
```
Start it with `./gradlew`

##### **Suggest a Fix**

Adding `background-color: black;` to `.navbar` element as seen in the picture fixed the issue.
![image](https://user-images.githubusercontent.com/35540617/57223546-1de6dc00-700f-11e9-8fb6-b4deec622a4b.png)

##### **JHipster Version(s)**

```
ordertracker@0.0.0 C:\development\order-tracker
+-- generator-jhipster@6.0.0-beta.0
`-- generator-jhipster-kotlin@0.7.0 -> C:\development\jhipster-kotlin
  `-- generator-jhipster@6.0.0-beta.0

```

##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**

<details>
<summary>.yo-rc.json file</summary>
<pre>
{
  "generator-jhipster-kotlin": {
    "promptValues": {
      "packageName": "eu.herbfoods.ordertracker"
    },
    "jhipsterVersion": "6.0.0-beta.0",
    "applicationType": "monolith",
    "baseName": "ordertracker",
    "packageName": "eu.herbfoods.ordertracker",
    "packageFolder": "eu/herbfoods/ordertracker",
    "serverPort": "8080",
    "authenticationType": "session",
    "cacheProvider": "ehcache",
    "enableHibernateCache": true,
    "websocket": false,
    "databaseType": "sql",
    "devDatabaseType": "h2Disk",
    "prodDatabaseType": "postgresql",
    "searchEngine": false,
    "messageBroker": false,
    "serviceDiscoveryType": false,
    "buildTool": "gradle",
    "enableSwaggerCodegen": false,
    "rememberMeKey": "YourJWTSecretKeyWasReplacedByThisMeaninglessTextByTheJHipsterInfoCommandForObviousSecurityReasons",
    "enableTranslation": true
  },
  "generator-jhipster": {
    "promptValues": {
      "nativeLanguage": "en"
    },
    "jhipsterVersion": "6.0.0-beta.0",
    "applicationType": "monolith",
    "baseName": "ordertracker",
    "clientFramework": "react",
    "useSass": true,
    "clientPackageManager": "npm",
    "testFrameworks": ["cucumber", "protractor"],
    "jhiPrefix": "jhi",
    "entitySuffix": "",
    "dtoSuffix": "DTO",
    "otherModules": [
      {
        "name": "generator-jhipster-kotlin",
        "version": "0.7.0"
      }
    ],
    "enableTranslation": true,
    "nativeLanguage": "en",
    "languages": ["en", "et", "de"],
    "blueprint": "generator-jhipster-kotlin",
    "blueprintVersion": "0.7.0",
    "clientTheme": "none"
  }
}

</pre>
</details>

##### **JDL for the Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

<details>
<summary>JDL entity definitions</summary>

<pre>
entity Product {
  name String required,
  ean String required,
  packageEan String required
}
entity OrderItem {
  quantity Integer
}
entity Order {
  orderNumber String,
  orderingDate LocalDate required,
  departureFromFactoryDate LocalDate,
  plannedArrivalDate LocalDate,
  actualArrivalDate LocalDate,
  orderStatus OrderStatus required,
  invoiceStatus InvoiceStatus required,
  invoiceNumber String
}
entity Buyer {
  name String required,
  email String required
}
entity Factory {
  name String required,
  email String required
}
entity Seller {
  name String required,
  email String required
}
entity Destination {
  name String required,
  address String required
}

enum OrderStatus {
  PLANNING,
  SUBMITTED,
  SELLER_CONFIRMED,
  FACTORY_ACCEPTED,
  PRODUCED,
  PICKED_UP,
  ARRIVED
}

enum InvoiceStatus {
  NOT_CREATED,
  SENT,
  PAID
}

relationship OneToOne {
  Buyer{user} to User,
  Factory{user} to User,
  Seller{user} to User
}
relationship ManyToOne {
  OrderItem{order} to Order,
  OrderItem{product} to Product,
  Order{destination} to Destination,
  Order{buyer} to Buyer,
  Order{seller} to Seller,
  Order{factory} to Factory
}

</pre>
</details>

##### **Environment and Tools**

java version "1.8.0_201"
Java(TM) SE Runtime Environment (build 1.8.0_201-b09)
Java HotSpot(TM) 64-Bit Server VM (build 25.201-b09, mixed mode)

git version 2.14.1.windows.1

node: v10.15.3

npm: 6.1.0

yeoman: 2.0.6

##### **Browsers and Operating System**

using Chrome browser on Windows 10

-   [ x ] Checking this box is mandatory (this is just to show you read everything)

**Comment by vishal423** [comment]
I should be fixed on master and available in v6.0. Can you try and confirm?

~Vishal

**Comment by pascalgrimaud** [comment]
Yes, it has been fixed here https://github.com/jhipster/generator-jhipster/pull/9610
Thanks @vishal423 for the answer and the fix

As the Kotlin blueprint use JHipster v6.0.0-beta.0, it could explain the issue.
2 solutions for you:
- wait or help to upgrade the Kotlin blueprint to use JHipster v6.0.0
- apply the fix in your header.scss file to include this part: https://github.com/jhipster/generator-jhipster/blob/master/generators/client/templates/react/src/main/webapp/app/shared/layout/header/header.scss.ejs#L59-L96

I'm closing this, as it's already fixed
